packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -0,0 +1,281 @@
+
+### Related Components
+
+To allow end-users to enter a custom value, use a [**combobox**](/packages/nimble-components/src/combobox/specs/IxD.md).
+
+![ ](spec-images/select-alternative-combobox.png)
+
+> NOTE: SEE FUTURE CONSIDERATIONS FOR MULTI-SELECTION
+
+To allow end-users to multi-select menu items, use a [**tag picker**](/packages/nimble-components/src/tag-picker/specs/IxD.md).
+
+![ ](spec-images/select-alternative-picker.png)
+
+## Design
+
+### Configuration
+
+#### Label
+
+> NOTE: NEEDS VISUAL DESIGN FOR PLACEMENT
+
+The client-user should be able to specify whether they want the label to be positioned at the top or to the left of the select or not at all. Selects that don't have a label should still have an ARIA label.
+
+![ ](spec-images/select-6.png)
+
+#### Icons
+
+The client-user should be able to specify whether they want left icons on menu items or not. Icons show up in menu items to the left of text and are displayed as part of the value in the field button.
+
+![ ](spec-images/select-7.png)
+
+#### Groups
+
+> QUESTION: IS THE SL "VIEW" PICKER A SELECT (SHOULD IT BE)? WHAT ABOUT GROUPS WITHOUT LABELS, WHAT ABOUT A SELECTION GROUP VS. AN ACTION GROUP OF MENU ITEMS (A.K.A MEGA-MENUS)?
+
+The client-user should be able to create groups for menu items in the menu. Groups are not interactable. Filtering by group name should show all the options under that group, whether or not it is included in the option text.
+
+![ ](spec-images/select-8.png)
+
+#### Secondary Text
+
+> NOTE: NEEDS VISUAL DESIGN
+
+The client-user should be able to specify whether they want secondary text on menu items or not.
+
+![ ](spec-images/select-9.png)
+
+The client-user should be able to specify whether they want other information (including groups) displayed as secondary text in the field input.
+
+![ ](spec-images/select-10.png)
+
+#### Selection
+
+> NOTE: NOT SURE THIS IS A REAL USE CASE FOR SELECT COMPONENT, SEE FUTURE CONSIDERATIONS FOR MULTI-SELECTION
+
+The client-user should be able to specify whether they want to display selection with or without a check mark.
+
+![ ](spec-images/select-11.png)
+
+#### Filter
+
+The client-user should be able to specify whether they want filtering available in the select component. The filter field appears in the menu.
+
+![ ](spec-images/select-12.png)
+
+### Behavior
+
+#### Minimum Width
+
+> NOTE: NEEDS DEV REVIEW
+
+The field button minimum width is three times the height of the component (to make components targetable on touch screens). The menu width is the same as the field button until reaching the minimum width (without truncating) of the longest string in the menu items.
+
+![ ](spec-images/select-13.png)
+
+#### Menu Height
+
+The menu height should be enough to display five to seven menu items before overflowing with a vertical scrollbar (not pictured).
+
+#### Overflow Text
+
+> NOTE: NEEDS DEV & VISUAL DESIGN REVIEW
+
+All text in the select component should truncate, not wrap.
```

Comment:
I'm a bit confused by this screenshot along with the guidance above that "Ellipses are not needed". Could you provide some clarification?

Reply:
Ellipses are not needed for the placeholder, unless long enough to truncate. I've added language to clarify this in the "Field Button" section.